lib/units/dimensions.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:get/get.dart';
+
+class Dimensions {
+  static double screenHeight = Get.context!.height;
+  static double screenWidth = Get.context!.width;
```

Comment:
Nên lấy tỉ lệ ở ngoài màn hình thì tốt hơn, đừng làm dụng 1 cái thư viện to chỉ để làm cái này